examples/create-react-app-material-ui/src/registerServiceWorker.js

```diff
@@ -0,0 +1,102 @@
+// In production, we register a service worker to serve assets from local cache.
+
+// This lets the app load faster on subsequent visits in production, and gives
+// it offline capabilities. However, it also means that developers (and users)
+// will only see deployed updates on the "N+1" visit to a page, since previously
+// cached resources are updated in the background.
+
+// To learn more about the benefits of this model, read https://goo.gl/KwvDNy.
+// This link also includes instructions on opting out of this behavior.
+/* @flow */
```

Comment:
Do I need to ignore this file in .flowconfig? It is giving me errors when I run `npm run flow` and seems to be unnecessary work since this file belongs to the create-react-app project.